.github/workflows/ci-actions.yml

```diff
@@ -99,6 +99,9 @@ jobs:
         with:
           name: maven-repo
           path: maven-repo.tgz
+      - name: Delete Local Artifacts From Cache
```

Comment:
I wonder if this shouldn't run before the `Cache Maven Repository` action :thinking:

Reply:
No, it needs to run after it. 

The cache action is kinda odd, as it adds a step at the end to actually save the result. Basically the initial step restores the cache if it exists, and the 'post' step after the build saves the result.